src/ecosystem.md

```diff
@@ -0,0 +1,24 @@
+# The SORA Ecosystem
+
+SORA is a community-first network within the Substrate ecosystem. As
+of writing, the SORA Kusama parachain is currently active, with the
+SORA Polkadot parachain coming soon.
+Community members and developers are the key stakeholders of SORA,
+therefore feature development is mapped out on the [SORA Integrated Plan](integrated-plan.md), which is open to receive milestone contributions through the RFP
+program.
+
+## What You Will Find in This Section
+
+The SORA Integrated Plan as well as it's current milestones are hosted
```

Comment:
```suggestion
The SORA Integrated Plan as well as its current milestones are hosted
```

Reply:
Grammatically speaking, its is ok for this one.

Reply:
`its` is correct, but you have `it's`, which is `it is` and doesn't work in this sentence: `it is current milestones are hosted here` doesn't make sense